tutorial/local-setup.md

```diff
@@ -35,58 +41,62 @@ PDF, Man, Epub, and others, including PHP which is what we use at php.net.
 
 ### Install PhD
 ```
-$ cd /tmp
+$ cd phpdoc
 $ git clone https://github.com/php/phd.git
 $ php phd/render.php --help
 ```
 
 ### Use PhD to build the documentation
 ```
-$ cd /tmp/git/doc-en
+$ cd phpdoc
 $ php doc-base/configure.php
-$ php /tmp/phd/render.php --docbook /tmp/git/doc-en/doc-base/.manual.xml --package PHP --format php
-$ cd /tmp/git/doc-en/mydocs/php-web/
+$ php phd/render.php --docbook doc-base/.manual.xml --package PHP --format php
+$ cd output/php-web
```

Comment:
Nit: not sure this needs to be here.  The prose can mention where the files are plopped.